cob_bringup/robots/raw3-1.xml

```diff
@@ -58,7 +59,7 @@
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="sim" value="$(arg sim)"/>
 		</include>
-		<include file="$(find cob_bringup)/components/legacy_schunk_powercube.launch">
+		<include file="$(find cob_bringup)/components/canopen_generic.launch">
```

Comment:
this won't work without proper configuration/yaml files...I guess this hasn't been tested on HW...